blog/june-2019-workshop.html

```diff
@@ -0,0 +1,275 @@
+
+  <br><br>
+
+  We're also providing native support for applying labeling functions using Spark and Dask,
+  making it easier to work with massive data sets.
+  Spark integration in particular has been a common feature request, and this was echoed
+  by the workshop attendees.
+
+  <h3>General purpose engine for data set creation</h3>
+
+  We redesigned Snorkel in version 0.9 to be a general purpose engine
+  for programmatically building and managing training data sets across different domains
+  and ML problem types.
+  TODO: EXTRA SENTENCE HERE?
+  We’re keeping the core programmatic labeling functionality from the current information
+  extraction application version, but removing the extraction-specific tooling and database
+  from the central library.
+  Workshop attendees were excited about this change, as they could see how to apply Snorkel to their
+  ML use cases - whether they used text, image, or tabular data - in a straightforward way
+  and integrate with their existing data infrastructure.
+
+  <br><br>
+
+  Many of our workshop participants also expressed their interest in continued support of the
+  current information extraction application version of Snorkel.
+  We’ll absolutely be doing this, hosted in a new GitHub repo and upgraded with the new
+  label model from the central Snorkel library.
+  Our information extraction system is just one example of a Snorkel-powered
+  end-to-end application.
+  We'll be releasing a host of other tutorials and examples in the next couple months.
+  They’ll have a new home in a dedicated tutorials repo, where you'll be able to see Snorkel
+  applied to a wide variety of ML problems.
+
+  <h3>A modern Python library</h3>
+
+  Workshop participants were excited to see modern installation support with pip and conda, as well
+  as lots and lots of unit and integration testing.
+  We also completed full API documentation - requested by everyone at the workshop - during a sprint
+  this past week.
+  All of this will make it easier to get started with, receive support, and contribute to Snorkel.
+
+  <br><br>
+
+  Stay tuned for the full Snorkel v0.9 release post, where we’ll share more details and resources!
+
+
+  <h2 id="workshop-contents">Overview of Workshop Sessions</h2>
+
+  The workshop was structured around the three core Snorkel operations: labeling,
+  transforming, and slicing.
+  Labeling functions were covered on the first day, and transformation and slicing
+  functions covered on the second day.
+  We also explored code from our state-of-the-art SuperGLUE submission,
+  demonstrating transformation and slicing on several of the tasks.
+  The first half of each day was divided between into presentations (both high-level and
+  technical or theoretical deep dives for those interested) and Jupyter notebook-based
+  tutorials.
+  We had two hours of open "office hours"-style sessions in the afternoon during which
```

Comment:
Is it worth mentioning some interests around video, causality etc. to give more structure to the open hours.can do without too